warehouse/templates/packaging/detail.html

```diff
@@ -20,6 +20,12 @@
   <header class="detail-header">
     <h1>{{ project.name }} <span class="version">{{ release.version }}</span></h1>
     {% if release.summary %}<p>{{ release.summary }}</p>{% endif %}
+    {% if release.version != all_releases[0].version %}
+    <div class="alert alert-warning">
+      {{ _('The latest version of this package is')
```

Comment:
This probably needs to be something like:

```
{{ _('The latest version of this package is <a href="%(version_link)s">%(version)s</a>', version_link=request.route_path('packaging.project', name=release.name), version=all_releases[0].version) }}
```

I *think* this will enable translators to put the translated content anywhere. You might also need a ``|safe()`` filter on the end of that to make it work.